robot_ws/src/voice_interaction_robot_msgs/msg/KeyValue.msg

```diff
@@ -0,0 +1,2 @@
+string key
```

Comment:
Why can't we get this message type from lex_common_msgs? https://github.com/aws-robotics/lex-ros2/blob/master/lex_common_msgs/msg/KeyValue.msg
It looks like it's been released to dashing http://packages.ros.org/ros2/ubuntu/dists/bionic/main/binary-amd64/Packages